scale/README.md

```diff
@@ -0,0 +1,46 @@
+JupyterHub Kubernetes Autoscaler
+===================================
+
+### Definitions
+
+**Critical Service =** Pods whose names start with `hub, proxy, statsd`
```

Comment:
We probably shouldn't be calling these 'Service', since that has a different and [very specific meaning](https://kubernetes.io/docs/user-guide/services/) in Kubernetes. 'pod name patterns' or something like that maybe?